resolwe_bio/processes/plots/bamliquidator.yml

```diff
@@ -0,0 +1,215 @@
+      required: False
+    - name: black_list
+      label: Black list
+      type: list:basic:string
+      required: False
+      description: >
+        One or more chromosome patterns to skip during bin liquidation. Default is to skip any
+        chromosomes that contain any of the following substrings chrUn _random Zv9_ _hap.
+    - name: threads
+      label: Threads
+      type: basic:integer
+      default: 1
+      description: >
+        Number of threads to run concurrently during liquidation.
+  output:
+    - name: analysis_type
+      label: Analysis type
+      type: basic:string
+      hidden: true
+    - name: output_dir
+      label: Output directory
+      type: basic:file
+    - name: counts
+      label: Counts HDF5 file
+      type: basic:file
+    - name: matrix
+      label: Matrix file
+      type: basic:file
+      required: False
+      hidden: "analysis_type != 'region'"
+    - name: summary
+      label: Summary file
+      type: list:basic:file:html
+      required: False
+      hidden: "analysis_type != 'bin'"
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+
+      re-save analysis_type {{ analysis_type }}
+
+      {% if regions_gtf and regions_bed %}
+        re-error "Please provide only one type of region file gff or bed file, accordingly to the analysis type."
+      {% endif %}
+
+      {% if regions_gtf and bin_size %}
+        re-error "Please provide only gff file or bin size, accordingly to the analysis type."
+      {% endif %}
+
+      {% if regions_bed and bin_size %}
+        re-error "Please provide only bed file or bin size, accordingly to the analysis type."
+      {% endif %}
+
+      if [{{ analysis_type }} == "bin" ] && [ {{ regions_bed }} ] || [ {{ regions_gtf }} ]; then
+        re-error "In Bin mode, regions bed or gff files can not be used. Please select different analysis type."
+      fi
+
+      if [{{ analysis_type }} == "region" ] && [ {{ regions_bed }} ] || [ {{ bin_size }} ]; then
+        re-error "In Region mode, bed file or bin size can not be used. Please select different analysis type."
+      fi
+
+      if [{{ analysis_type }} == "bed" ] && [ {{ regions_gtf }} ] || [ {{ bin_size }} ]; then
+        re-error "In BED mode, region gtf file or bin size can not be used. Please select different analysis type."
+      fi
+
+      BAM_PARAMS=()
+      BAM_BAI_PARAMS=()
+      ALL_MODE_PAR=()
+
+      BAM_PARAMS+=({% for bam_file in bam %}"{{bam_file.bam.file}}" {% endfor %})
+      BAM_BAI_PARAMS+=({% for bai_file in bam %}"{{bai_file.bai.file}}" {% endfor %})
+      SKIP_PLOT='{% if skip_plot %}--skip_plot {% endif %}'
+      BLACK_LIST='{% if black_list %}--black_list {{black_list|join(' ')}} {% endif %}'
+      SENSE='{% if sense %}--sense {{ sense }} {% endif %}'
+      EXTENSION='{% if extension %}-e {{ extension }} {% endif %}'
+
+      ALL_MODE_PAR+=("{{ cell_type }}" -n {{threads}} --xml_timings ${SKIP_PLOT} ${BLACK_LIST} ${EXTENSION} ${SENSE} -o output)
```

Comment:
Change this to string.

Does it make sense to include `SKIP_PLOT` for all modes if it is also relevant for `bin` mode?